src/ci/docker/run.sh

```diff
@@ -152,28 +153,28 @@ else
     exit 1
 fi
 
-mkdir -p $HOME/.cargo
-mkdir -p $objdir/tmp
-mkdir -p $objdir/cores
+mkdir -p "$HOME/.cargo"
+mkdir -p "$objdir/tmp"
+mkdir -p "$objdir/cores"
 mkdir -p /tmp/toolstate
 
-args=
+args=()
 if [ "$SCCACHE_BUCKET" != "" ]; then
-    args="$args --env SCCACHE_BUCKET"
-    args="$args --env SCCACHE_REGION"
-    args="$args --env AWS_ACCESS_KEY_ID"
-    args="$args --env AWS_SECRET_ACCESS_KEY"
+    args+=(--env SCCACHE_BUCKET)
+    args+=(--env SCCACHE_REGION)
+    args+=(--env AWS_ACCESS_KEY_ID)
+    args+=(--env AWS_SECRET_ACCESS_KEY)
 else
-    mkdir -p $HOME/.cache/sccache
-    args="$args --env SCCACHE_DIR=/sccache --volume $HOME/.cache/sccache:/sccache"
+    mkdir -p "$HOME/.cache/sccache"
+    args+=(--env "SCCACHE_DIR=/sccache" --volume "$HOME/.cache/sccache:/sccache")
```

Comment:
This doesn't need quotes, it doesn't have variables.
```suggestion
    args+=(--env SCCACHE_DIR=/sccache --volume "$HOME/.cache/sccache:/sccache")
```